mlir/include/mlir/Dialect/XeGPU/IR/XeGPUOps.td

```diff
@@ -172,10 +202,16 @@ def XeGPU_CreateNdDescOp: XeGPU_Op<"create_nd_tdesc", [Pure, ViewLikeOpInterface
     /// and `const_shape` will be used to represent the shape of
     /// source operand. They overide static shape from source memref type.
     ArrayRef<int64_t> getStaticSizes() {
+      /// To be compatible with OffsetSizeAndStrideOpInterface, which expects valid return value and perform checks
+      static  llvm::SmallVector<int64_t, 4> emptyShape;
```

Comment:
I haven't fully considered this case yet but the fact we need to add a persistent (dummy or not) state looks like a red flag.